MATLAB/mex_CUDA_glnxa64.xml

```diff
@@ -1,73 +1,74 @@
-<?xml version="1.0" encoding="UTF-8" ?>
-<!-- Copyright 2013 The MathWorks, Inc. -->
-<config
-    Name="nvcc"
-    ShortName="nvcc"
-    Manufacturer="NVIDIA"
-    Version=""
-    Language="CUDA"
-    Priority="A"
-    Location="$NVCC" >
-    <Details
-        CompilerExecutable="$CXX"
-        CompilerDefines="$DEFINES"
-        CompilerFlags="$CXXFLAGS"
-        OptimizationFlags="$CXXOPTIMFLAGS"
-        DebugFlags="$CXXDEBUGFLAGS"
-        IncludeFlags="$INCLUDE"
-        LinkerLibraries="$LINKLIBS"
-        LinkerOptimizationFlags="$LDOPTIMFLAGS"
-        LinkerDebugFlags="$LDDEBUGFLAGS"
-
-        CompilerDefineFormatter="-D%s"
-        LinkerLibrarySwitchFormatter="-l%s"
-        LinkerPathFormatter="-L%s"
-    />
-    <vars
-          CMDLINE1="$CXX -c $DEFINES $INCLUDE $NVCCFLAGS $CXXFLAGS $OPTIM $SRC -o $OBJ"
-          CMDLINE2="$LDXX $LDFLAGS $LDTYPE $LINKOPTIM $LINKEXPORT $OBJS $CXXLIBS $LINKLIBS -o $EXE"
-          CMDLINE3="rm -f $OBJ"
-          
-          CXX="$NVCC"
-          DEFINES="--compiler-options=-D_GNU_SOURCE,$MATLABMEX"
-          MATLABMEX="-DMATLAB_MEX_FILE"
-          NVCCFLAGS="-gencode=arch=compute_35,code=sm_35  -gencode=arch=compute_50,code=sm_50   -gencode=arch=compute_60,code=sm_60   -gencode=arch=compute_61,code=sm_61 -gencode=arch=compute_70,code=sm_70 -gencode=arch=compute_75,code=sm_75  --default-stream per-thread  $NVCC_FLAGS"
-          CXXFLAGS="--compiler-options=-ansi,-fexceptions,-fPIC,-fno-omit-frame-pointer,-pthread, -Dnullptr=0 "

```

Comment:
This one is still being replaced, which should not (`-Dnullptr=0`)

Reply:
Fixed in defa305